src/pages/Projects/components/RoleList/RoleList.tsx

```diff
@@ -72,6 +73,8 @@ export default function RoleList({ project }: RoleListProps): JSX.Element {
         roleToEdit={roleToEdit}
         setRoleToEdit={setRoleToEdit}
         updateRole={updateRole}
+        updateAssignment={updateAssignment}
+        destroyAssignment={destroyAssignment}
```

Comment:
would it be worth moving the `useAssignmentMutations` call into the component instead of using props?

Reply:
hmm I'm actually taking advantage of this for unit testing purposes - to supply mocked functions to the component. Would it be okay to keep it like this?